datm/cime_config/config_component.xml

```diff
@@ -39,6 +39,17 @@
     <desc>Name of atmospheric component</desc>
   </entry>
 
+  <entry id="DATM_MODE_ANOMALY_FORCING_MISMATCH">
```

Comment:
We talked about this offline and I'll leave it up to you whether this is worth having in place, since it's likely to not be used that often.

This does make it easy to do and clear on what it's doing though, which is good.

Reply:
Removed with 8aab1ae.